testutil/assertions/events.go

```diff
@@ -0,0 +1,85 @@
+package assertions
+
+import (
+	"fmt"
+
+	"github.com/stretchr/testify/assert"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// PrependToEach prepends the provided prefix to each of the provide lines.
+func PrependToEach(prefix string, lines []string) []string {
```

Comment:
This is pretty generic. Might be better to put this in general.

Reply:
I thought about doing that, but decided to leave it here since it technically isn't even related to unit testing other than being used by the event string creation stuff. Since it was written for that event stuff, I thought it was more at home with the rest of the event stuff.